tests/ui/single_component_use_path.fixed

```diff
@@ -0,0 +1,10 @@
+// run-rustfix
+// compile-flags: --edition 2018
+#![warn(clippy::single_component_use_path)]
+
+
+
+fn main() {
```

Comment:
I am not sure how to handle the space between warn and fn main. How to extend span to include the emtpy line before and after the import?

By the way [ci skip] doesn't seem working?

Reply:
Don't worry about this, this'll be handled by rustfmt. 

I didn't know [ci skip] was a thing? 😮

Reply:
Oh, in that case I can move on.

Well, with [CI skip] the pr won't trigger ci tests until [ci skip] is removed from title, I suppose.

Reply:
Is this a Github feature, because it's not something implemented in this repos CI?

Reply:
At least in Gitlab this works

Reply:
I have read it from some where in github, could be rust related, I am not sure. Probably not a github feature, I didn't find anything on github help.

Reply:
It's purely CI related and has nothing to do with GitHub or GitLab.

Reply:
Thanks for clarifying, @mati865.